code/src/main/java/com/onlinecontacttracing/servlets/CustomMessageAndContactsServlet.java

```diff
@@ -0,0 +1,82 @@
+package com.google.sps.servlets;
+
+
+import com.google.api.client.googleapis.auth.oauth2.GoogleIdTokenVerifier;
+import com.google.api.client.googleapis.auth.oauth2.GoogleIdToken.Payload;
+import com.google.api.client.googleapis.auth.oauth2.GoogleIdToken;
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.api.client.json.JsonFactory;
+import com.google.api.client.json.jackson2.JacksonFactory;
+
+import com.googlecode.objectify.annotation.Entity;
+import com.googlecode.objectify.annotation.Load;
+import com.googlecode.objectify.Objectify; 
+import com.googlecode.objectify.ObjectifyFactory; 
+import com.googlecode.objectify.ObjectifyService; 
+import static com.googlecode.objectify.ObjectifyService.ofy;
+
+import com.onlinecontacttracing.storage.CustomizableMessage;
+import com.onlinecontacttracing.storage.PositiveUserWithMessage;
+import com.onlinecontacttracing.messsaging.GeneratedUserId;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.lang.ClassCastException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.UUID;
+/**Servlet to put custom message data into Ofy store */
+@WebServlet("/customMessage")
+public class CustomMessageAndContactsServlet extends HttpServlet {
+  private final ArrayList<String> employeeComment = new ArrayList<String>();
+  private static final JsonFactory JSON_FACTORY = JacksonFactory.getDefaultInstance();
+  
+  /**
+   * Get the number of emails to display from the user
+   */
+  private int getNumEmails(HttpServletRequest request) {
+    // Get the input from the form.
+    String userNum = request.getParameter("number-of-recipients-box");
+    int requestedNum;
+    //Convert input into an int
+    try {
+      requestedNum = Integer.parseInt(userNum);
+    } catch (NumberFormatException e) {
+      System.err.println("Could not convert to int: " + userNum);
+      return -1;
+    }
+    return requestedNum;
+  }
+
+  /**
+   * Adds the Emails and custom message to the Objectify store
+   */
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    int numberOfRecipients = 0;
+    try {
+      numberOfRecipients = Integer.parseInt(request.getParameter("number-of-recipients-box"));
+    } catch(NumberFormatException e) {
+        System.out.println("Not a number");
```

Comment:
You use .err above and .out above. 

Also, did you intend to have only one place you convert numbers instead of doing the work twice?

Reply:
Can do and just the one taking care of it now